<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WNDASCLauncherMain</class>
 <widget class="QMainWindow" name="WNDASCLauncherMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>656</width>
    <height>297</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ASC Launcher</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_10">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_9">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6"/>
        </item>
        <item>
         <widget class="QTextEdit" name="edProtocol">
          <property name="font">
           <font>
            <family>Monospace</family>
           </font>
          </property>
          <property name="locale">
           <locale language="English" country="UnitedStates"/>
          </property>
          <property name="lineWrapMode">
           <enum>QTextEdit::NoWrap</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QProgressBar" name="pbCompleted">
            <property name="maximumSize">
             <size>
              <width>280</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="locale">
             <locale language="English" country="UnitedStates"/>
            </property>
            <property name="value">
             <number>24</number>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="locale">
             <locale language="English" country="UnitedStates"/>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btnRunMacro">
            <property name="maximumSize">
             <size>
              <width>120</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="locale">
             <locale language="English" country="UnitedStates"/>
            </property>
            <property name="text">
             <string>Run Macro</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
   <zorder></zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>656</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
